src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/publicApi/PublicLicencePolicyService.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.publicApi
+
+import jakarta.persistence.EntityNotFoundException
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.policy.LicencePolicy
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.policies.POLICY_V1_0
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.policies.POLICY_V2_0
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.policies.POLICY_V2_1
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licencePolicy.LicencePolicy as PublicLicencePolicy
+
+@Service
+class PublicLicencePolicyService(
+  private val policies: List<LicencePolicy> = listOf(
+    POLICY_V1_0,
+    POLICY_V2_0,
+    POLICY_V2_1,
+  ),
+) {
```

Comment:
think its ok to either pull in the other policy service here, or maybe just a static reference to the list of policies?, rather than declaring these twice - the set of policies from each endpoint should be consistent

Reply:
Done